projects/wave-core-new/src/lib/datatable/table/table.component.ts

```diff
@@ -33,10 +33,14 @@ export class DataTableComponent implements OnInit, AfterViewInit, OnDestroy, OnC
 
     readonly layerTypes = ResultTypes.VECTOR_TYPES;
 
+    selectedFeature$ = new BehaviorSubject<FeatureSelection>({feature: undefined});
+
     dataSource = new FeatureDataSource();
     displayedColumns: Array<string> = [];
+    featureColumns: Array<string> = [];
 
     protected layerDataSubscription: Subscription = undefined;
+    protected selectedLayerSubscription: Subscription = undefined;
```

Comment:
rename to selectedFeatureSubscription

Reply:
a9118f18b483cec650eb659b904b16d3f73d8fe2